### PR TITLE
Improve the look of the top bar of embedded sections. #90

### PR DESCRIPTION
**Pull request by swxsw**
Removed header textbox, replaced with grey text.
Changed close button of embedded handbook to transparent background, turns red when pointer hovering over.
staging site at https://rawgit.com/swxsw/2103-website/90-improve-embedded-top-bar/index.html

### CODE REVIEW

**Comment by acjh** [comment]
@swxsw Please mention `Fixes #90` in the description like [your previous PR](https://github.com/nus-cs2103/website/pull/207), so that:
- reviewers can quickly refer to the issue, and
- the issue is closed automatically when this PR is merged.

**Comment by acjh** [comment]
@swxsw Please also update the link to your staging site:
https://rawgit.com/swxsw/2103-website/90-improve-embedded-top-bar/

**Comment by damithc** [comment]
Some quick comments:
1. `X` is more suitable to indicate a 'close' button? `-` usually indicates 'minimize', am I right?
2. The title looks a bit too big? 
3. It's nicer if the title and the button can be inside a box that looks like a 'panel header' (similar to other panels in the schedule page) so that they are separated from the main content of the box.

**Comment by swxsw** [comment]
Noted with thanks, will make changes according later.

**Comment by swxsw** [comment]
The design should be roughly done, though one potential issue is that the embedded button is vertically aligned differently on firefox and chrome, firefox looks right but chrome renders it slightly too high

File: styles/common.css [hunk 1]
```diff
@@ -289,6 +289,30 @@ code {
...
 }
+/*
+ * embedded button
+ */
+
+.btn-dismiss-embedded {
+    background-color: transparent !important;
+    border-radius: 3px;
+    border: 1px solid;
+    color: #FFFFFF;
```
Comment (acjh) [inline styles/common.css]: I believe it's `color: white;`.

File: styles/common.css [hunk 2]
```diff
@@ -384,20 +408,31 @@ table.two-column-content td {
 }
 
 .embedded {
-    padding: 10px 10px;
-    border: 1px dashed #000;
+    padding: 0px 10px 10px 10px;
```
Comment (swxsw) [inline styles/common.css]: Noted and updated

File: styles/common.css (hunk 1 above)
Comment (swxsw) [inline styles/common.css]: Noted with thanks

File: styles/common.css [hunk 3]
```diff
@@ -289,6 +289,30 @@ code {
     background-color: #ffff00 !important;
     color: #000 !important;
 }
+/*
+ * embedded button
+ */
+
+.btn-dismiss-embedded {
+    background-color: transparent !important;
```
Comment (swxsw) [inline styles/common.css]: Understood, changed accordingly

File: styles/common.css [hunk 4]
```diff
@@ -289,6 +289,30 @@ code {
     background-color: #ffff00 !important;
     color: #000 !important;
 }
+/*
```
Comment (swxsw) [inline styles/common.css]: Removed

File: styles/common.css [hunk 5]
```diff
@@ -384,20 +409,31 @@ table.two-column-content td {
...
+}
+
+.embeddedHeading {
+    background-color: transparent;
+    color: #FFFFFF;
+    font-size: 1 em;
+    font-weight: normal;
+    text-align: center;
+    vertical-align: middle;
     width: 100%;
```
Comment (acjh) [inline styles/common.css]: Only `color: white` appears to be necessary.

File: styles/common.css [hunk 6]
```diff
@@ -384,20 +409,31 @@ table.two-column-content td {
...
 
 .embedded {
-    padding: 10px 10px;
-    border: 1px dashed #000;
+    padding: 0px 10px 10px 10px;
+    border: 1px solid #7A629B;
+    border-radius: 3px;
     margin: 10px auto;
     background-color: #efefef;
+    position: relative;
```
Comment (acjh) [inline styles/common.css]: No need for `position: relative`, since there is no `left`, `right`, `top` or `bottom`.

File: styles/common.css [hunk 7]
```diff
@@ -291,6 +291,31 @@ code {
...
+
+.btn-dismiss-embedded {
+    background-color: transparent;
+    border-radius: 3px;
+    border: 1px solid;
+    color: white;
+    float: right;
+    margin-top: -2px;
+    padding: 1px 12px 1px 12px;
+
```
Comment (acjh) [inline styles/common.css]: Remove unnecessary newline.

File: styles/common.css (hunk 6 above)
Comment (swxsw) [inline styles/common.css]: Noted with thanks, rather new to css. Thanks for the tips

File: styles/common.css (hunk 7 above)
Comment (swxsw) [inline styles/common.css]: Done:)

File: styles/common.css (hunk 5 above)
Comment (swxsw) [inline styles/common.css]: I see, thanks

File: styles/common.css (hunk 6 above)
Comment (acjh) [inline styles/common.css]: You're welcome! I'm learning and experimenting along the way as well :)

File: styles/common.css (hunk 2 above)
Comment (acjh) [inline styles/common.css]: This was left unchanged?

Comment (swxsw) [inline styles/common.css]: By shorthand, do you mean `padding:  0px 10px 10px;` in this case?

Comment (acjh) [inline styles/common.css]: Nope, I mean `padding: 0 10px 10px 10px`.

Comment (swxsw) [inline styles/common.css]: Should be done in the latest commit

File: scripts/common.js [hunk 8]
```diff
@@ -34,9 +34,9 @@ function pullContent(fileName, elementSelector, title, sectionName) {
     $(elementSelector).load(toBeLoaded, function(response, status, xhr) {
         if (status == 'success') {
             $(elementSelector).addClass('embedded');
-            $(elementSelector).prepend('<div><span class="embeddedHeading">' + title + '</span><button onclick="$(\'' + elementSelector + '\').html(\'\');' +
+            $(elementSelector).prepend('<div><div id = "embedded-heading-container"><span class="embedded-heading">' + title + '</span><button onclick="$(\'' + elementSelector + '\').html(\'\');' +
```
Comment (acjh) [inline scripts/common.js]: Remove spaces around `=` sign.

File: styles/common.css [hunk 9]
```diff
@@ -384,23 +402,26 @@ table.two-column-content td {
...
+    left: -10px;
+    padding: 7px 10px 7px 10px;
+    position: relative;
     width: 100%;
 }
 
+.embedded-heading {
+    color: white;
+}
+
```
Comment (acjh) [inline styles/common.css]: Shift `.embedded-heading` back above `#embedded-heading-container`.

**Comment by acjh** [comment]
Looks good. @damithc Any comments?

**Comment by damithc** [comment]
Looks good. 👍 Can go ahead and merge.
I don't think we need to make the background of the box gray, but that can be a separate issue. @acjh can create a new issue for that?

**Pull request merged**